[2025-06-14T22:32:27.472+0100] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-14T22:32:27.478+0100] {taskinstance.py:2066} INFO - Dependencies not met for <TaskInstance: customer_churn_training_dag.train_on_small_dataset manual__2025-06-14T21:30:20.409329+00:00 [failed]>, dependency 'Task Instance State' FAILED: Task is in the 'failed' state.
[2025-06-14T22:32:27.480+0100] {local_task_job_runner.py:163} INFO - Task is not able to be run
